trunk/NDHMS/Routing/module_RT.F

```diff
@@ -837,6 +838,20 @@ subroutine LandRT_ini(did)
                     nlst(did)%reservoir_observation_lookback_hours, &
                     nlst(did)%reservoir_observation_update_time_interval_seconds)
 
+          type is (rfc_forecasts)
+              call reservoir%init(                              &
+                    rt_domain(did)%RESHT(lake_index),           &
+                    rt_domain(did)%HRZAREA(lake_index),         &
+                    rt_domain(did)%WEIRH(lake_index),           &
+                    rt_domain(did)%WEIRC(lake_index),           &
+                    rt_domain(did)%WEIRL(lake_index),           &
+                    rt_domain(did)%ORIFICEE(lake_index),        &
+                    rt_domain(did)%ORIFICEC(lake_index),        &
+                    rt_domain(did)%ORIFICEA(lake_index),        &
```

Comment:
This may need to be updated to support the new dam_length parameter. Please either a) confirm it doesn't, or b) add support.

Reply:
b) add support. 
Thanks

Reply:
dam_length parameter has been added.